modules/Core/pages/minecraft/banner.php

```diff
@@ -49,16 +49,7 @@
         $cache->setCache('banner_cache_' . urlencode($server->name));
         if (!$cache->isCached('image')) {
             // Internal or external query?
-            $query_type = DB::getInstance()->get('settings', ['name', 'external_query'])->results();
-            if (count($query_type)) {
-                if ($query_type[0]->value == '1') {
-                    $query_type = 'external';
-                } else {
-                    $query_type = 'internal';
-                }
-            } else {
-                $query_type = 'internal';
-            }
+            $query_type = Util::getSetting('external_query') ? 'external' : 'internal';
```

Comment:
Was this changed in a recent PR so that we now have `query_type` instead of `external_query` or am I imagining things?

Reply:
You're right: https://github.com/NamelessMC/Nameless/pull/3244/files#diff-ddc3a26d3f3aee9e751be4668fd9c35c5e593001adc6ebc7d5c1259713cdc116

I wonder why it didn't produce conflicts

Reply:
> I wonder why it didn't produce conflicts

Mystery solved, this branch was just very outdated